zenoh/src/lib.rs

```diff
@@ -351,7 +351,11 @@ pub mod time {
 
 /// Configuration to pass to [`open`] and [`scout`] functions and associated constants
 pub mod config {
-    pub use zenoh_config::{WhatAmI, WhatAmIMatcher};
+    pub use zenoh_config::{EndPoint, Locator, WhatAmI, WhatAmIMatcher, ZenohId};
```

Comment:
why putting back `EndPoint` and `Locator`?

Reply:
They can't really be used from the stable API, having them there is superfluous I would say

Although users will write Endpoints in their configuration and they would benefit from the docs, it's still odd having a public item that once constructed can't be used in any way.

Reply:
`Locator` is part of the `Hello` and scouting API. And they are stable.